integrations/analytics.js-integration-tiktok-pixel/lib/index.ts

```diff
@@ -0,0 +1,23 @@
+import Integration from '@segment/analytics.js-integration';
+
+const TIKTOK_PIXEL_GLOBAL = 'ttq';
```

Comment:
Just curious of the significance of the `q` in this? I would have expected `ttp` or `ttg` or `ttpg`

Reply:
I actually have no idea. That's how they name the global variable to store this object in all of their documentation.

Facebook's pixel uses `fbq`, Twitter ... you guessed it - `twq` - but couldn't not find any information about the significance of `q` at all!

Reply:
Well, Googling was no help - but ChatGPT to the rescue.

> Tracking pixels often have the letter 'q' at the end of their variable names as a convention to indicate that the variable is a queue or a list of tracking pixels to be loaded.

Reply:
Oh! Nice work ChatGPT